src/routes/Settings/routes/Profile/components/Profile.jsx

```diff
@@ -27,74 +25,66 @@ class Profile extends React.Component {
     const name = target.name;
 
     this.setState({
-      [name]: value
+      [name]: value,
     });
   }
 
   handleSubmit(event) {
     // This prevents a '?' from being appended to the URL
     event.preventDefault();
-    var data = {};
 
-    if(this.props.firstName !== this.state.firstName) {
-      data.firstName = this.state.firstName;
+    if (this.props.name !== this.state.name) {
```

Comment:
Just a small optimization. I don't call the function unless the name has changed (`this.props.name` comes from redux, `this.state.name` is used to store local changes), that way I'm not hitting the server with useless requests.